dev-docs/workflows/testing.md

```diff
@@ -0,0 +1,21 @@
+# Test
+
+## Unit tests
+
+Running unit tests with Bazel:
+
+```sh
+bazel test //...
+```
+
+## Integration tests
+
+You can run all integration like this:
+
+```sh
+ctest -j `nproc`
+```
+
+You can limit the execution of tests to specific targets with e.g. `ctest -R integration-node-operator`.
+
+Some of the tests rely on libvirt and won't work if you don't have a virtualization capable CPU. You can find instructions on setting up libvirt in our [QEMU README](qemu.md).
```

Comment:
Integration tests are not yet migrated. We have AB#3067 to track this. The reason this wan't done yet is that this can be a lot of work to do properly and gives us no direct benefits (the tests do work and run in CI today, they are just not very hermetic).